code-challenges.test.js

```diff
@@ -47,6 +68,38 @@ const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
 const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
 // Expected output: ["Temperature", "Database", "Chopsticks", "Mango"]
 
+describe("capitalLetters") = () => {
+    it("return strings with the first letter capitalized", () => {
+        expect(capitalLetters(randomNouns1)).toEqual("Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew")
+        expect(capitalLetters(randomNouns2)).toEqual("Temperature", "Database", "Chopsticks", "Mango")
+    })
+
+}
```

Comment:
This test looks better, but your test should contain all the info it needs inside the scope {} of the describe. Which means your test variables should live inside the test.